phpcs.xml.dist

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="SilverStripe">
+	<description>CodeSniffer ruleset for SilverStripe coding conventions.</description>
+
+	<file>src</file>
+	<file>tests</file>
+
+    <!-- Show progress and output sniff names on violation, and add colours -->
+    <arg value="p" />
+    <arg name="colors" />
+    <arg value="s" />
+
+	<!-- base rules are PSR-12 -->
+	<rule ref="PSR12" >
+		<!-- Current exclusions -->
+		<!--
+        <exclude name="PSR1.Methods.CamelCapsMethodName" />
+		<exclude name="PSR1.Files.SideEffects.FoundWithSymbols" />
+		<exclude name="PSR2.Classes.PropertyDeclaration" />
+		<exclude name="PSR2.ControlStructures.SwitchDeclaration" /> <!-- causes php notice while linting -->
```

Comment:
Will clean this up before marking ready-to-merge.

Reply:
Should clean it up then as this PR's in review?

Reply:
Yes! Sorry. I thought I'd forget that's why I added that comment lol.

Reply:
Cleaned up, and resolved linting failures from a local run.